meta-balena-common/recipes-core/initrdscripts/files/resindataexpander

```diff
@@ -56,6 +56,10 @@ resindataexpander_run() {
 	        sync
                 # We return when we've done the expansion
 		info "Data partition at $datapart expanded."
+		# Expand filesystem
+		info "resindataexpander: Expand ext4 filesystem on ${datapart}"
+		resize2fs -f "${datapart}"
+		sync
```

Comment:
Can't we pull and use the existing `resin-filesystem-expand` here? Looks like we would be doing the same thing in two places.

Reply:
I think we can probably remove resin-filesystem-expand from the main image as it serves no purpose. It needs some cross-device testing to make sure and that was too distracting to do in this PR, but I will do it as a separate one.

Reply:
https://github.com/balena-os/meta-balena/tree/alexgg/resindataexpander